model/jobrole.ts

```diff
@@ -0,0 +1,6 @@
+export class JobRole {
+    jobRole?: string
```

Comment:
[Just To Note] - Its best to keep your branch focused. I.e. We shouldn't really see Capability or Specification in this branch.